### ISSUE TITLE
Contrast in HeaderBar

### ISSUE THREAD

**Issue by cassidyjames** [TenderOwl/Norka#21]
It looks like the contrast in your headerbar does not meet WCAG A contrast guidelines (at least 3:1 contrast). I would recommend picking a lighter HeaderBar color, or setting the foreground color and text shadow in CSS to ensure it is contrast-compliant.

**Comment by amka** [comment]
Thanks for your suggestion, definitely it's true! Text shadow doesn't come to my mind. Will check for it and update it with the next release.
